test/gtest/ucm/cuda_hooks.cc

```diff
@@ -92,16 +92,16 @@ class cuda_hooks : public ucs::test {
     }
 
 
-    void check_mem_alloc_events(void *ptr, size_t size) {
+    void check_mem_alloc_events(void *ptr, size_t size, int expect_mem_type) {
```

Comment:
minor: maybe declare default value for `expect_mem_type` as `UCM_MEM_TYPE_CUDA`. Then you would not need to add it in the old tests.

Reply:
fixed